cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/connect/class-api.php

```diff
@@ -228,18 +231,20 @@ function ( $item ) use ( $transformation_index ) {
 	 * @return string
 	 */
 	public function cloudinary_url( $public_id, $args = array(), $size = array(), $clean = false ) {
+
 		$defaults = array(
 			'resource_type' => 'image',
 			'version'       => 'v1',
 		);
 		$args     = wp_parse_args( array_filter( $args ), $defaults );
 		// Correct Audio to Video.
-		if ( 'audio' === $args['resource_type'] ) {
-			$args['resource_type'] = 'video';
-		}
+		$args['resource_type'] = $this->covert_resource_type( $args['resource_type'] );
+
 		// check for version.
 		if ( ! empty( $args['version'] ) && is_numeric( $args['version'] ) ) {
 			$args['version'] = 'v' . $args['version'];
+		} else {
```

Comment:
@DavidCramer 

The `$default` variable already sets the `v1`.

Reply:
This is why I asked about it on the other PR. I knew I put it back somewhere 😂